### PR TITLE
Fix paragraph / line # alignment issue

### PR DESCRIPTION
**Pull request by nedredmond**
## Summary:
Switch from `className` to `id` in css selectors for passage wrappers in an effort to gain higher priority than article paragraph styles.

Change `line-height` on passage `em` elements to 0 since the italicized characters have a subtly different size. See second video for demo of how this resolves the problem.

https://github.com/Khan/perseus/assets/23404711/ae4902b7-128c-45ad-b2f6-56195852b50f

https://github.com/Khan/perseus/assets/23404711/7de0aaeb-891c-4a03-a97a-fa9f147d4892

Issue: LC-698

## Test plan:
- Test in webapp with npm

### CODE REVIEW

**Comment by nedredmond** [comment]
Made PR to test solution in webapp, since it has always looked fine in perseus

**Comment by nedredmond** [comment]
Fixed for [passage article](https://www.khanacademy.org/internal-courses/test-everything/test-everything-2-without-mastery/te-passage/a/passage-article), not yet for LSAT.

File: packages/perseus/src/widgets/passage.tsx [hunk 1]
```diff
@@ -512,9 +512,9 @@ export class Passage extends React.Component<PassageProps, PassageState> {
 
         return (
             <div>
-                <div className="perseus-widget-passage-container">
+                <div id="perseus-widget-passage-container">
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/passage.tsx:515]: 🤔 I see how this fixes the issue, but I'm concerned about moving to id's here as we could (maybe theoretically) have more than one passage widget on the page (yielding multiple elements with the same id, which is not allowed).

> The id [global attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes) defines an identifier (ID) which must be unique in the whole document.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

**Review by jeremywiebe — CHANGES_REQUESTED** [review]
Hmm... I think we'll need to find a different solution because of multiple identical ids being possible on the page.

File: packages/perseus/src/widgets/passage.tsx (hunk 1 above)
Comment (nedredmond) [reply packages/perseus/src/widgets/passage.tsx:515]: @jeremywiebe Yeah, I thought about that, but I don't think _practically_ that we'd ever encounter it. Plus even if we did, it'd be a warning, not a show-stopping issue.

It doesn't fix the issue for the test prep passages anyway, so this issue is still open. I'll keep exploring.

**Comment by nedredmond** [comment]
@jeremywiebe , whenever you have time, this is ready for re-review

**Review by jeremywiebe — APPROVED** [review]
Cool. Thanks for changing the id's to be unique! One last thing I thought of. We might want to ensure that `joinString()` also ensures that we don't use non-allowed characters from the title in the `id` attribute.  It looks like the allowed characters are defined by `ident` on this page (https://www.w3.org/TR/CSS21/grammar.html#scanner).  The simple approach could just strip out all non-alpha chars and then join.

**Pull request merged**